packages/record-data/addon/-private/relationships/state/relationship.ts

```diff
@@ -509,11 +515,11 @@ export default class Relationship {
         const id = guidFor(inverseRecordData);
 
         if (this._hasSupportForImplicitRelationships(inverseRecordData) && seen[id] === undefined) {
-          const relationship = implicitRelationshipStateFor(inverseRecordData, this.inverseKeyForImplicit)
+          const relationship = implicitRelationshipStateFor(inverseRecordData, this.inverseKeyForImplicit);
```

Comment:
😱 I'm surprised we didn't have this issue lint warnings?!

Reply:
We have a PR from @HeroicEric in the works to fix lint, turns out we weren't linting any TS files